scripts/test/lint.py

```diff
@@ -94,6 +94,7 @@ class IncludesRequiredFieldsOnly(Lint):
         "description",
         "authors",
         "dependencies",
+        "tags",
```

Comment:
Please remove this change and add it after all packages have the category in the nuspec.